main.yml

```diff
@@ -349,11 +347,32 @@ rhel:
   top_level: true
 
 settings:
-  title: Cloud Services Settings
+  title: Settings
+  top_level: true
+  isBeta: true
```

Comment:
Sorry I noticed this a little late, but `isBeta` is a field that only exists under `api`. Check out how it's used in other apps, like Approval, for instance.